cmd/minikube/cmd/config/enable.go

```diff
@@ -42,12 +42,8 @@ var addonsEnableCmd = &cobra.Command{
 			exit.WithError("enable failed", err)
 		}
 		if addon == "dashboard" {
-			out.T(out.Waiting, "Some dashboard features require the 'metrics-server' add-on")
-			err = addons.SetAndSave(ClusterFlagValue(), "metrics-server", "true")
-			if err != nil {
-				exit.WithError("enable failed", err)
-			}
-			out.T(out.AddonEnable, "The 'metrics-server' add-on was enabled automatically")
+			out.T(out.Tip, "Some dashboard features require the 'metrics-server' add-on. For full features consider enabling metrics-server addon.")
```

Comment:
nit: add-on should be addon

Also, I would tell the users how to enable it:

```
Some dashboard features require the metrics-server addon. To enable all features please run:

minikube addons enable metrics-server
```

Reply:
thanks updated